app/views/volunteers/edit.html.erb

```diff
@@ -51,6 +51,12 @@
                         class: "btn btn-outline-success" %>
           <% end %>
         <% end %>
+        <% if current_user.supervisor? && @volunteer.invitation_accepted_at == nil%>
```

Comment:
We want this to be supervisor **or admin** actually but this is a good start! :)